src/gui/syncrunfilelog.cpp

```diff
@@ -145,7 +145,6 @@ void SyncRunFileLog::logItem( const SyncFileItem& item )
 
     const QChar L = QLatin1Char('|');
     _out << ts << L;
-    _out << QString::number(item._requestDuration) << L;
```

Comment:
**Note:** This changes the format of the log, if anybody was parsing it assuming the position of fields, this might break it. I'm assuming that we don't support a stable format for this file.